docs/classic/release-notes.md

```diff
@@ -12,6 +12,13 @@ The Plan-B deprecation should not affect any current users of the CTI Adapter, a
 
 **Important:** when upgrading the CTI Adapter, please make sure that the Salesforce Lambdas are also updated to the newest version.
 
+## 5.19 April 2022
+
+- **Enhancement:** replace call recording audio streaming via cloudfront distribution with the connect native get-recording endpoint. This change makes it so that the cloudfront infrastructure and associated setup process is no longer necessary. Please note that this change will remove audio recording infrastructure from your AWS account, please make sure to test this change before fully deploying.
```

Comment:
Could someone elaborate on this? I don't see any `get-recording` APIs in Amazon Connect
https://docs.aws.amazon.com/connect/latest/APIReference/Welcome.html

Reply:
it's a "hidden api" method

assuming your connect login user has the correct permissions and call recording is enabled, you can use the following path to stream audio:
`/connect/get-recording?format=wav&callLegId=[contactId]`